src/GrpcProtos/capture.proto

```diff
@@ -323,6 +341,7 @@ message ProducerCaptureEvent {
     // numbers starting with 16.
     //
     // Please keep these alphabetically ordered.
+    ApiEvent api_event = 10;
```

Comment:
I do not think it matters, we are saving 1 byte by using low number but on the other hand the structure of the proto wasting a lot of them, not just absence of variant encoding for most fields we also pass unnecessary padding over the wire. I think this is one case I would argue this does not make sense. We better off saving low number for compact protos.

Reply:
Ok, let's change it back then, thanks.